pkg/cmd/workspace/code.go

```diff
@@ -131,10 +133,29 @@ func selectWorkspaceProject(workspaceId string, profile *config.Profile) (*strin
 	return nil, errors.New("no projects found in workspace")
 }
 
+func openTerminalSsh(workspaceId string, projectName string, profile config.Profile) error {
```

Comment:
@harkiratsm this works great!

I would just suggest moving this function to the `ide` package so we are consistent.

We understand that `ide` might not be the best classification for "Terminal SSH" but we'll revisit that in the future.

Reply:
Got it, moving function to the `ide` package.